paper.md

```diff
@@ -1,33 +1,31 @@
 ---
-title: 'Separation using ChatGPT'
+title: 'BS-RoFormer: The SAMI-ByteDance Music Source Separation System for Sound Demixing Challenge 2023'
 tags:
-  - separation
-  - u-net
+  - music source separation
+  - transformer
+  - rotary position embedding
+  - band-split
 authors:
-  - name: Fabian-Robert Stoter^[co-first author] # note this makes a footnote saying 'co-first author'
-    orcid: 0000-0003-0872-7098
-    affiliation: "1, 2" # (Multiple affiliations must be quoted)
-  - name: Author Without ORCID^[co-first author] # note this makes a footnote saying 'co-first author'
-    affiliation: 2
-  - name: Author with no affiliation^[corresponding author]
-    affiliation: 3
+  - name: Ju-Chiang Wang^[co-first author]
+    orcid: 0009-0002-8265-4229
+    affiliation: 1 
+  - name: Wei-Tsung Lu^[co-first author] # note this makes a footnote saying 'co-first author'
+    affiliation: 1
+  - name: Qiuqiang Kong
+    orcid: 0000-0003-2864-0475
+    affiliation: 1
+  - name: Yun-Ning Hung
+    orcid: 0000-0002-7242-6903
+    affiliation: 1
 affiliations:
- - name: Audioshake
+ - name: SAMI, ByteDance Inc.
    index: 1
- - name: Institution Name
-   index: 2
- - name: Independent Researcher
-   index: 3
-date: 10 September 2023
+date: 26 September 2023
 bibliography: paper.bib
-arxiv-doi: 10.21105/joss.01667
+arxiv-doi: 10.48550/arXiv.2309.02612
```

Comment:
@asriverwang we decided to not use the arxiv link on the side. So please reference your paper inline in the abstract so that it appears as a reference after the abstract

Reply:
sure. I have added the reference inline in the abstract.